netbout-web/src/main/java/com/netbout/rest/MarkdownTxtmark.java

```diff
@@ -47,10 +47,18 @@
         // @checkstyle LineLengthCheck (1 line)
         "(?<!\\]\\()(?<!=\")(https?:\\/\\/[a-zA-Z0-9-._~:\\?#@!$&'*+,;=%\\/]+[a-zA-Z0-9-_~#@$&'*+=%\\/])(?![\\w.]*\\]\\()"
     );
+    /**
+     * Pattern to detect lines which should have a line break on the end.
+     */
+    private static final Pattern NEW_LINE = Pattern.compile(
+        "^ {0,3}\\S* ?$", Pattern.MULTILINE
```

Comment:
@mkordas you're right about a like with spaces inside
I fixed regexp and changed the test as well